content/ecosystem/hosting/providers.toml

```diff
@@ -48,3 +48,14 @@ description = """
 
 Ossrox is a company from Germany dedicated to hosting open-source services. Not only has the company name been derived from the motto "Open-Source Software Rocks", but also their corporate culture. The three most important principles of the company are security, privacy and sustainability - thanks to open-source software. In addition to Matrix, Ossrox also provides many other solutions e.g. in the messaging, groupware and web meeting segments.
 """
+
+[[providers]]
+name = "Federated Computer"
+image = "federated.png"
+website = "https://federated.computer"
+operating_since = "2022"
+description = """
+- Hosted Homeservers
+
+Federated Computer provides Element and Matrix hosting in the context of a team platform including integrated SSO. For $39/m>
```

Comment:
The description seems a bit mangled, can you make sure it's complete :)?

Reply:
Thanks for the comment. I clarified the description. Best.